src/server/main_service.cc

```diff
@@ -590,6 +600,42 @@ void Service::Init(util::AcceptServer* acceptor, std::vector<facade::Listener*>
   GenericFamily::Init(&pp_);
   server_family_.Init(acceptor, std::move(listeners), &cluster_family_);
 
+  if (string flag = GetFlag(FLAGS_replicaof); !flag.empty()) {
+    string ip, port;
+    {
+      auto pos = flag.find_last_of(':');
+      CHECK(pos != string::npos) << "Invalid format for --replicaof: missing ':'.";
```

Comment:
We could set a default port, I guess most installations use it?

Reply:
That would, however, be an issue when supporting IPv6, so perhaps that's not a good suggestion..

Reply:
well, for IPv6 we require starting the string with `[` so technically this can be implemented. I'm happy to do either

Reply:
Coming back to this. Is there a conventional default port? Maybe 6378?